docs/src/userguide/language_basics.rst

```diff
@@ -430,14 +699,16 @@ An external C++ function that may raise an exception can be declared with::
 
     cdef int spam() except +
 
+.. note:: Pure python mode currently does not have equivalent to `except +`.
```

Comment:
Created issue for that #4282

Reply:
```suggestion
.. note:: Pure python mode currently does not have equivalent to `except +`. (Github issue :issue:`4282`)
```

Reply:
And also it would be really great if you will add those references to all other notes that needs that.

Reply:
As said in #4282 we should not allow it, and it should be reflected in the text + this conversation should be resolved